test/service.js

```diff
@@ -64,26 +64,26 @@ test('returns non-base-type', function t(assert) {
 
 test('service extends from another service', function t(assert) {
     assert.deepEqual(
-        Object.keys(thrift.services.Qux.functionsByName),
+        Object.keys(thrift.Qux),
```

Comment:
This is also a backward compatibility issue. The `thrift.services` index is no longer from name to model, but directly to the surface, in this case functionsByName. The ThriftService model is available via `thrift.models.Qux`.